From 7fc78783018cf2ce27f8e6829f9eff1d9032541c Mon Sep 17 00:00:00 2001

From: Peter Hutterer <peter.hutterer@who-t.net>

Subject: [PATCH] config: don't hardcode /etc for 10-evdev.conf

Use sysconfdir instead to avoid amusements like /usr/etc/ and the like.

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---

 config/Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)


diff --git a/config/Makefile.am b/config/Makefile.am
index 675a3b2..58286b8 100644
--- a/config/Makefile.am
+++ b/config/Makefile.am
@@ -9,7 +9,7 @@ AM_CFLAGS += $(UDEV_CFLAGS)
 libconfig_la_SOURCES += udev.c
 libconfig_la_LIBADD = $(UDEV_LIBS)

-xorgconfddir = $(datadir)/X11/$(XF86CONFIGDIR)
+xorgconfddir = $(sysconfdir)/$(XF86CONFIGDIR)
 xorgconfd_DATA = 10-evdev.conf

 else
